backend/services/mock.js

```diff
@@ -2,6 +2,36 @@ const { User, Role, ApplicationPipeline } = require("../models");
 
 async function createMockData() {
   // Roles
+  const alum_role = new Role({
+    name: "Alum",
+    permissions: {
+      roster: false,
+      recruitment: false,
+      admin: false,
+      final_approval: false,
+    },
+  });
+  await alum_role.save();
+  const unassigned_role = new Role({
+    name: "Unassigned",
+    permissions: {
+      roster: true,
+      recruitment: false,
+      admin: false,
+      final_approval: false,
+    },
+  });
+  await unassigned_role.save();
+  const designer_role = new Role({
+    name: "Designer",
+    permissions: {
+      roster: true,
+      recruitment: true,
+      admin: false,
+      final_approval: false,
+    },
+  });
+  await designer_role.save();
   const public_role = new Role({
```

Comment:
Not sure what David's naming conventions were, but since you added roles maybe we should update these to match your naming conventions? For exmpale: developer_role